src/Server/Services/Analytics/AnalyticsService.lua

```diff
@@ -659,22 +928,18 @@ end
 		amount = 100
 	})
 	```
-	
+
 	@param data ResourceEvent
-	@return { [any]: any }
+	@return Promise
 ]=]
-function AnalyticsService:LogResourceEvent(data: ResourceEvent): { [any]: any }
+function AnalyticsService:LogResourceEvent(data: ResourceEvent)
```

Comment:
```suggestion
function AnalyticsService:LogResourceEvent(data: ResourceEvent): {[any]: any}
```
Outside of a custom defined promise type, {[any]: any} is the closest to a return type for a promise in my opinion.